routing/routing.xml

```diff
@@ -614,12 +624,22 @@
 			<select value="1.3" t="highway" v="steps"/>
 			<select value="3" t="route"   v="ferry"/>
 			<select value="5" t="bicycle" v="dismount"/>
-			<select value="22" t="bicycle" v="designated"/>
-			<if param="avoid_unpaved">
-				<select value="4" t="highway" v="track"/>
-				<select value="4" t="highway" v="footway"/>
-				<select value="3" t="highway" v="path"/>
-			</if>
+			<!-- Line above: I now think it would be a better solution
```

Comment:
If we remove bicycle=designated here, will OsmAnd route via main highway it this case (with "Fast bike" driving style)?